language/tools/cost-synthesis/src/stack_generator.rs

```diff
@@ -192,11 +191,7 @@ where
     // Certain operations are only valid if their values come from module-specific data. In
     // particular, CreateLibraAccount. But, they may eventually be more of these as well.
     fn points_to_module_data(&self) -> bool {
```

Comment:
Seems like we should be able to delete this + do something simpler at each call site.

Reply:
I was leaving it for @tzakian to make the call but honestly you are right, that is somewhat useless at this point and could be removed. Having said so, @tzakian please comment if you feel otherwise

Reply:
also I have removed this, so @tzakian let me know if you want it there